ui/vic-ui-h5c/vic/src/vic-app/src/app/summary-view/summary-view.component.ts

```diff
@@ -97,8 +98,10 @@ export class VicSummaryViewComponent implements OnInit, OnDestroy {
 
         this.refreshSubscription = this.refreshService
             .refreshObservable$.subscribe(() => {
-                console.log('root info is being refreshed');
-                this.fetchRootInfo();
+                this.zone.run(() => {
```

Comment:
I thought you were not having to use zone any longer wasn't that the point of this update?